src/navigations/TabNavigation/TabNavigation.tsx

```diff
@@ -12,16 +12,35 @@ import HomeScreenV2 from '../../screens/HomeScreen/HomeScreenV2';
 import { useSelector } from 'react-redux';
 import { AllTaskScreenStack } from '../../screens/Stacks/AllStack';
 import { scale } from '../../utils/utils';
+import CalendarInviteScreen from '../../screens/Calendar/CalendarInviteScreen';
 
 const tab = createBottomTabNavigator();
 
+const renderLabel = ({ focused }) => {
+  return (
+    <Text
+      style={{
+        fontSize: Fonts.Tab_Text_Font,
+        color: focused ? Colors.Tab_Active_Color : Colors.Tab_Inactive_Color,
+      }}
+    >
+      {Strings.Tab_Calendar}
+    </Text>
+  );
+};
+
+const renderImage = () => {
+  return <Image style={TabViewStyle.tab_icon} source={Images.calendar} />;
```

Comment:
Is this one-line abstraction useful?

Reply:
yes ,to render icon, but this is also going to use for other tabs too

Reply:
commited